load-test/scenarios/teacher-edit-pupil/teacher-edit-pupil.jmx

```diff
@@ -0,0 +1,398 @@
+hmap.put(2, &quot;C&quot;);
+hmap.put(3, &quot;D&quot;);
+hmap.put(4, &quot;E&quot;);
+hmap.put(5, &quot;F&quot;);
+hmap.put(6, &quot;G&quot;);
+hmap.put(7, &quot;H&quot;);
+hmap.put(8, &quot;J&quot;);
+hmap.put(9, &quot;K&quot;);
+hmap.put(10, &quot;L&quot;);
+hmap.put(11, &quot;M&quot;);
+hmap.put(12, &quot;N&quot;);
+hmap.put(13, &quot;P&quot;);
+hmap.put(14, &quot;Q&quot;);
+hmap.put(15, &quot;R&quot;);
+hmap.put(16, &quot;T&quot;);
+hmap.put(17, &quot;U&quot;);
+hmap.put(18, &quot;V&quot;);
+hmap.put(19, &quot;W&quot;);
+hmap.put(20, &quot;X&quot;);
+hmap.put(21, &quot;Y&quot;);
+hmap.put(22, &quot;Z&quot;);
+
+String upnNuunber= hmap.get(checkLetterIndex) + upnDigits;
+vars.put(&quot;UPN_NUMBER&quot;,upnNuunber);
+</stringProp>
+          </BeanShellPreProcessor>
+          <hashTree/>
+        </hashTree>
+        <DebugSampler guiclass="TestBeanGUI" testclass="DebugSampler" testname="Debug Sampler" enabled="false">
+          <boolProp name="displayJMeterProperties">false</boolProp>
+          <boolProp name="displayJMeterVariables">true</boolProp>
+          <boolProp name="displaySystemProperties">false</boolProp>
+        </DebugSampler>
+        <hashTree/>
+        <ResultCollector guiclass="ViewResultsFullVisualizer" testclass="ResultCollector" testname="View Results Tree" enabled="false">
+          <boolProp name="ResultCollector.error_logging">false</boolProp>
+          <objProp>
+            <name>saveConfig</name>
+            <value class="SampleSaveConfiguration">
+              <time>true</time>
+              <latency>true</latency>
+              <timestamp>true</timestamp>
+              <success>true</success>
+              <label>true</label>
+              <code>true</code>
+              <message>true</message>
+              <threadName>true</threadName>
+              <dataType>true</dataType>
+              <encoding>false</encoding>
+              <assertions>true</assertions>
+              <subresults>true</subresults>
+              <responseData>false</responseData>
+              <samplerData>false</samplerData>
+              <xml>false</xml>
+              <fieldNames>true</fieldNames>
+              <responseHeaders>false</responseHeaders>
+              <requestHeaders>false</requestHeaders>
+              <responseDataOnError>false</responseDataOnError>
+              <saveAssertionResultsFailureMessage>true</saveAssertionResultsFailureMessage>
+              <assertionsResultsToSave>0</assertionsResultsToSave>
+              <bytes>true</bytes>
+              <sentBytes>true</sentBytes>
+              <threadCounts>true</threadCounts>
+              <idleTime>true</idleTime>
+              <connectTime>true</connectTime>
+            </value>
+          </objProp>
+          <stringProp name="filename">resultsTree.csv</stringProp>
+        </ResultCollector>
+        <hashTree/>
+        <JSR223Listener guiclass="TestBeanGUI" testclass="JSR223Listener" testname="JSR223 Listener" enabled="false">
```

Comment:
what is this for? its name doesnt tell me anything about it

Reply:
This is some thing to log the error codes and their response messages while debugging. I have disabled it to make it an optional.